vercel.json

```diff
@@ -1430,6 +1430,10 @@
     {
       "source": "/docs/guides/upgrade-guides/upgrade-from-prisma-1/upgrading-the-prisma-layer-postgres",
       "destination": "/docs/guides/upgrade-guides/upgrade-from-prisma-1/upgrading-the-prisma-layer-postgresql"
+    },
+    {
+      "source": "/docs/concepts/components/prisma-schema/generators#community-generators",
```

Comment:
Can you redirect specific anchors on a page?

Reply:
I removed this. I forgot to push the commit